rebar.config.script

```diff
@@ -0,0 +1,6 @@
+case erlang:function_exported(rebar3, main, 1) of
+    true -> CONFIG;
+    false -> 
+        %% remove plugin for rebar2 builds... cause doh!
+        lists:keydelete(plugins, 1, CONFIG)
```

Comment:
I don't guess we will ever need to add other plugins to this repo, but just in case, this seems more specific and less likely to cause unintended consequences:

```erlang
{plugins, Plugins} = lists:keyfind(plugins, 1, CONFIG),
lists:keystore(plugins, 1, CONFIG, {plugins, lists:delete(pc, Plugins)})
```